src/admin/admin.proto

```diff
@@ -15,6 +15,8 @@ message ClusterInfo {
 
   string proxy_host = 5;
   bool proxy_tls = 6;
+
+  bool paused = 7; // True if this pachd is in "paused" mode.
```

Comment:
It appears that for the comment to propagate to the python SDK it needs to be on the line above the field.

Reply:
I'll change that.

Reply:
Done.